app/containers/MusicProvider/MusicDetails/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ *
+ * MusicDetails
+ *
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Card } from 'antd';
+import { injectIntl } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { selectMusicData, selectRoutePath } from '../selectors';
+import styled from 'styled-components';
+import { improveImg } from '@components/StyledCard';
+
+const StyleDiv = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+`;
+const Para = styled.p`
+  font-family: 'Lucida Console', 'Courier New', monospace;
```

Comment:
we shouldn't be switching font-family's. we should ideally just have one font-family throughout the application that is set in the global-styles.js